extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcClientBuildTimeConfig.java

```diff
@@ -1,16 +1,18 @@
 package io.quarkus.grpc.runtime.config;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
 
+@ConfigMapping(prefix = "quarkus.grpc.client")
```

Comment:
I was surprised by the import change above. Shouldn’t it be grpc-client with a dash to be backward compatible?

Reply:
> I was surprised by the import change above. 

Why? 

> Shouldn’t it be grpc-client with a dash to be backward compatible?
Ah good catch, you're right: https://quarkus.io/guides/all-config#quarkus-grpc_quarkus-grpc-client-stork-proactive-connections